### PR TITLE
docs: customize envoyproxy dualstack

### PR DESCRIPTION
**Pull request by juwon8891**
## What's Changed
- Added documentation for customizing EnvoyProxy IP family configuration
- Added note about cluster requirements for DualStack networking

### CODE REVIEW

File: site/content/en/docs/tasks/operations/customize-envoyproxy.md [hunk 1]
```diff
@@ -1003,6 +1003,48 @@ spec:
 {{% /tab %}}
 {{< /tabpane >}}
 
+## Customize EnvoyProxy IP Family
+
+You can customize the IP family configuration for EnvoyProxy via the EnvoyProxy Config.
```
Comment (arkodg) [inline site/content/en/docs/tasks/operations/customize-envoyproxy.md:1008]:
```suggestion
You can customize the IP family configuration for EnvoyProxy via the EnvoyProxy Config.
This allows the Envoy Proxy fleet to serve external client over IPv4 as well as IPv6.

The below configuration sets the `ipFamily` to `DualStack` to allow ingressing IPv4 as well as IPv6 traffic.

**Note**: Envoy Gateway relies on the [Service](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services) spec of the BackendRef resource (linked to xRoutes) to decide which type of IP addresses to use to route to them.
```

**Comment by arkodg** [comment]
thanks @juwon8891, added a comment
can you also update the same doc in `v1.2/` and `latest/`

File: site/content/en/docs/tasks/operations/customize-envoyproxy.md [hunk 2]
```diff
@@ -1003,6 +1003,53 @@ spec:
...
+This allows the Envoy Proxy fleet to serve external client over IPv4 as well as IPv6.
```
Comment (arkodg) [inline site/content/en/docs/tasks/operations/customize-envoyproxy.md:1009]:
```suggestion
This allows the Envoy Proxy fleet to serve external clients over IPv4 as well as IPv6.
```

**Comment by arkodg** [comment]
@juwon8891 can you address the CI failures

**Comment by juwon8891** [comment]
> @juwon8891 can you address the CI failures

I resolved problem!

**Review by arkodg — APPROVED** [review]

**Review by zirain — APPROVED** [review]

**Pull request merged**